checks/s3/check_test.go

```diff
@@ -0,0 +1,16 @@
+package s3
+
+import (
+	"context"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestNewWithBucketError(t *testing.T) {
+	check := New(Config{
+		Region: "usa-east-1",
+	})
+
+	err := check(context.Background())
+	require.Error(t, err)
+}
```

Comment:
would be better if we could test the successful check, either mocking or spinning and S3 compatible API like minio

Reply:
Yeah. Agreed. I'll update it.